src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -101,7 +101,7 @@ public PublicClientApplicationBuilder WithIosKeychainSecurityGroup(string keycha
         /// </summary>
         /// <param name="enableBroker"></param>
         /// <returns></returns>
```

Comment:
missing stuff

Reply:
oops! 

---
In reply to: [300211740](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1251#discussion_r300211740) [](ancestors = 300211740)